<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">	/*Cambiamos el color de la ventana*/
	#MainWindow, #centralwidget{
		background-color: #F4F6F0;
	}

	/*Estilos para el bot√≥n*/
	QPushButton{
		background-color: white;
		border-radius: 0px;
	    border-width: 2px;
        border-style: solid;
        border-color: #485550;
		color: black;
		font-family: 'Roboto';
		font-size: 17px;
	}

	QPushButton:hover{
		background-color:#C0EB6A;
		cursor: pointer;
		color: black;
	}

		QPushButton{
		background-color: #485550;
		border-radius: 10px;
	    border-width: 2px;
        border-style: solid;
        border-color: black;
		color: white;
		font-family: 'Roboto';
		font-size: 17px;

	}

		#Output{
		background-color: white;
		border-radius: 5px;
	    border-width: 3px;
        border-style: solid;
        border-color: #485550;
		color: #485550;
		font-family: 'Roboto';
		font-size: 17px;
	}


	QPlainTextEdit,QTextEdit{
		background-color: white;
		border-radius: 5px;
	    border-width: 3px;
        border-style: solid;
        border-color: #485550;
		color: #485550;
		font-family: 'Roboto';
		font-size: 16px;
	}
	
		#label{
		
		border-radius: 10px;
	    border-width: 0px;
        border-style: solid;
        border-color: #485550;
		color: black;
		font-family: 'Roboto';
		font-size: 30px;
	}
	</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnGenerar">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>GENERAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>881</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CONJUNTO SIGUIENTE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>P -&gt; Ab | +cD | -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INPUT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>VARIABLES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TERMINALES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OUTPUT</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtOutput">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>20</width>
      <height>261</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtVariables">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>A,B,C,D,E,</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtTerminales">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>+,*,/,then,else</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtInicial">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>P</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INICIAL</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
